src/main/java/frc/robot/subsystems/ArmConfig.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems;
+
+import frc.robot.Config;
+
+public class ArmConfig {
```

Comment:
The ArmConfig class should really be a nested class in Config. Last year we had them separate when we shouldn't have.